src/electionguard/decryption/coordinator.c

```diff
@@ -152,6 +155,13 @@ Decryption_Coordinator_receive_share(Decryption_Coordinator c,
         }
     }
 
+    //free 
+    for (uint32_t i = 0; i < share_rep.num_tallies; i++)
```

Comment:
we don't need these values anymore so we free them

Reply:
Is it guaranteed that these values are either null (i.e. in case of error) or allocated?

Reply:
they are allocated at the beginning of the function with `Crypto_encryption_rep_new` which is a wrapper around gmp's `mpz_init`.